test/domain/relation/establish.spec.ts

```diff
@@ -1,14 +1,16 @@
 
 import { describe, expect, it } from 'vitest';
-import { CREATOR0, CREATOR1, REQUESTER1, REQUESTER2, RelationAlreadyExists, establish } from './_fixtures/relation.fixture';
+import { CREATOR0, CREATOR1, QUERY0, QUERY1, RECORD_TYPE_CREATOR, RECORD_TYPE_RELATION, REQUESTER1, REQUESTER2, RelationAlreadyExists, UNKNOWN_REQUESTER, database, establish } from './_fixtures/relation.fixture';
 
 describe('domain/relation/establish', () =>
 {
     it('should establish a relation', async () =>
     {
-        const relation = await establish(REQUESTER2, CREATOR0);
+        await establish(REQUESTER2, CREATOR0);
+        const query = QUERY0;
```

Comment:
This line can be removed if the query definition is used in the `findRecord` directly.